sherpa/models/basic.py

```diff
@@ -1959,7 +1999,76 @@ def calc(self, p, xlo, xhi=None, **kwargs):
                                       p, xlo, xhi, **self.otherkwargs)
 
 
+# DOC note: we do not expose Integrator1D by default so it is not
+# mentioned as a See Also link here.
+#
 class Integrate1D(RegriddableModel1D):
+    """Integrate a model across each bin (one dimensional).
+
+    Numerically integrate a one-dimensional model across each bin of a
+    "histogram" dataset - that is one with low and high edges for each
+    bin. The model to be integrated is supplied as an input to this
+    model and any attribute changes must be made before this.
+
+    Attributes
+    ----------
+    epsabs
+        The maximum absolute difference allowed when integrating the
+        model. This parameter is always frozen.
+    epsrel
+        The maximum relative difference allowed when integrating the
+        model. This parameter is always frozen.
+    maxeval
+        The maximum number of iterations allowed per bin.  This
+        parameter is always frozen.
+
+    Notes
+    -----
+    If `imdl` is an instance of `Integrate1D` and `omdl` the model to
+    integrate, then `imdl(omdl)` creates the integrated form. Note
+    that changes to the `Integrate1D` parameters - such as `epsabs` -
+    must be made before creating this integrated form.
+
+    Examples
+    --------
+
+    The Gauss1D is used as an example here since it - as can most Sherpa
+    models - already be used with an histogram dataset. The evaluation
```

Comment:
So the point of using this is to compare the build-in with the "by-hand" integration? Maybe add "in the example we compare the integration build into Gauss1D with the integration done by Integrate1D, but in practice, one would use Integrate1D only on models that do not define integration over bins themselves." just to really drive this point home.

Reply:
i had to think about it, so other users of Sherpa might have to, too.

Reply:
I wanted an easy-to-write example, but most of our models already support integration, so this was the best I could come up with on short notice. Then I ran out of energy to explain my thinking ;-)

Reply:
I've reworked this example and it's hopefully better now.

Reply:
I have already approved it, so I can't approve it again. Looks good to me.